main/modes/picross/mode_picross.c

```diff
@@ -812,14 +1005,26 @@ void drawPicrossScene(display_t* d)
                 drawSinglePixelFromWSG(d,i,j,&p->selectedLevel->completedWSG);
             }
         }
+
+        //Draw the title of the puzzle, centered.
+        //We Should get the y position from the 0 y position 
+        // uint16_t y = p->drawScale + p->topPad - p->UIFont.h;//this is y1 of boxFromCoords solved for position 0, then shifted up the height of the font.
+        //I don't know if I want the title drawn right above the puzzle or just at the top. at the top for now.
+
+        //TODO: still not quite centered.
+        int t = strlen(p->selectedLevel->title);//get character count of title
```

Comment:
There's a function called `textWidth()` which you give a font and a string and it returns the pixel width of the string. It's better than `strlen()` and a guess at the per-char width.

Reply:
Thanks! Easy fix with, now knowing that function exists.

Reply:
Knowing is half the battle!